config/routes.rb

```diff
@@ -16,7 +16,12 @@
         get :friends, to: 'users/friends#index'
       end
 
-      resources :games, only: [:create]
+      # Games crud
+      resources :games do
+        post :games, to: 'games/games#create'
+      end
```

Comment:
I think we want to keep the `resources :games, only: [:create]` route here and get rid of

```
 resources :games do
   post :games, to: 'games/games#create'
end
```

That seemed to be creating a lot of routes that we didn't need unless I'm missing something?

Reply:
I wasn't sure which one was best! Sorry about that! I'll edit and get that pushed back up in just a bit.

Reply:
Updated!